pkgs/os-specific/linux/roccat-tools/default.nix

```diff
@@ -19,10 +19,16 @@ stdenv.mkDerivation rec {
       /return/c \
         return g_build_path("/", g_get_user_data_dir(), "roccat", NULL);
     }' libroccat/roccat_helper.c
+
+    substituteInPlace udev/90-roccat-kone.rules \
+      --replace "/bin/sh" "${runtimeShell}" \
+      --replace "/sbin/modprobe" "${kmod}/bin/modprobe" \
+      --replace "/bin/echo" "${coreutils}/bin/echo"
   '';
 
   nativeBuildInputs = [ cmake pkg-config gettext ];
   buildInputs = [ dbus dbus-glib libgaminggear libgudev lua ];
+  runtimeDeps = [ coreutils kmod ];
```

Comment:
```suggestion
```
I am not sure if that line does anything.

Reply:
I just tested locally without this line and the udev rules still appear to work, so I have pushed this change.